plugins/headless/hu.bme.mit.gamma.headless.server/resources/gamma-wrapper.yaml

```diff
@@ -162,7 +162,7 @@ paths:
                 $ref: '#/components/schemas/ProjectFile'
         '404':
           description: Log file does not exist.
-  /gamma/loglevel:
+  /gamma/log/level:
```

Comment:
It is **setLogLevel** `/gamma/loglevel` in the docs, should it not be changed too?